src/woo_publications/metadata/api/serializer.py

```diff
@@ -27,6 +28,43 @@ class Meta:  # type: ignore
         }
 
 
+class OrganisationSerializer(serializers.ModelSerializer):
+    class Meta:  # type: ignore
+        model = Organisation
+        fields = ("uuid", "identifier", "naam", "oorsprong", "is_actief")
+        read_only_fields = (
+            "uuid",
+            "identifier",
+            "oorsprong",
+        )
+        extra_kwargs = {
+            "naam": {
+                "required": False,
+                "help_text": _(
+                    "The name of the organisation (can only be modified when `oorsprong` is `{custom_entry}`)."
+                ).format(custom_entry=OrganisationOrigins.custom_entry),
+            },
+            "is_actief": {
+                "required": False,
+            },
+        }
+
+    def update(self, instance, validated_data):
+        if (
+            validated_data.get("naam")
+            and instance.oorsprong != OrganisationOrigins.custom_entry
+        ):
+            raise serializers.ValidationError(
+                {
+                    "naam": _(
+                        "You cannot modify the name of organisations populated from a value list."
+                    )
+                }
+            )
```

Comment:
actually - taking a second look at this, why is this not done in `def validate(self, attrs): ...`?